live-examples/js-examples/array/array-foreach.js

```diff
@@ -2,6 +2,7 @@ const array1 = ['a', 'b', 'c'];
 
 array1.forEach(element => console.log(element));
 
-// expected output: "a"
-// expected output: "b"
-// expected output: "c"
+// expected output:
+// "a"
```

Comment:
Hi @pnevares!
Currently, I am working on unit tests. They will test the real output against the expected ones. Unfortunately, I couldn't find a better solution than adding the "expected output" prefix to each line. So I think we should leave them now as they are.

Reply:
Thanks for the info! I can restore the prefixes in this PR shortly.

Reply:
This part of the PR should be reverted!